js-apps/meep-frontend/src/js/containers/exec/network-characteristics-event-pane.js

```diff
@@ -131,14 +131,21 @@ class NetworkCharacteristicsEventPane extends Component {
     super(props);
 
     this.state = {
-      dialogOpen: false,
-      currentElementType: ''
+      dialogOpen: false
+      // currentElementType: ''
```

Comment:
You may need to add a state for the element type select menu. Other option would be to filter menu options to list only supported types.

Reply:
Added code to update the element type list based on scenario deployed